AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/preview/actions.js

```diff
@@ -304,6 +329,9 @@ define([
                                                 fileName = filePathArray[imageIndex]
                                                  .substring(0, filePathArray[imageIndex].lastIndexOf('.'));
                                             }
+                                            $.ajaxSetup({
```

Comment:
I don't think license and save requests should be async - the process ends with locate functionality that may fail if user will navigate through UI in meantime

Reply:
I found a random bug, when we click confirm the popup need to be closed and not wait until save will execute, because then the modal triggered to close, but popup still wait to response from save method, and after, we have overlay from this confirmation popup that not closed, so i add async to close popup of confirmation immediately